webpack.config.js

```diff
@@ -205,16 +205,47 @@ module.exports = [
         },
         performance: getPerformanceHints(5 * 1024)
     }),
+
+    // Because both video-blur-effect and rnnoise-processor modules are loaded
+    // in a lazy manner using the loadScript function with a hard coded name,
+    // i.e.loadScript('libs/rnnoise-processor.min.js'), webpack dev server
+    // won't know how to properly load them using the default config filename
+    // and sourceMapFilename parameters which target libs without .min in dev
+    // mode. Thus we change these modules to have the same filename in both
+    // prod and dev mode.
     Object.assign({}, config, {
         entry: {
             'video-blur-effect': './react/features/stream-effects/blur/index.js'
         },
         output: Object.assign({}, config.output, {
             library: [ 'JitsiMeetJS', 'app', 'effects' ],
-            libraryTarget: 'window'
+            libraryTarget: 'window',
+            filename: '[name].min.js',
+            sourceMapFilename: '[name].min.map'
         }),
         performance: getPerformanceHints(1 * 1024 * 1024)
     }),
+
+    Object.assign({}, config, {
+        entry: {
+            'rnnoise-processor': './react/features/stream-effects/rnnoise/index.js'
+        },
+        node: {
+            // Emscripten generated glue code "rnnoise.js" expects node fs module,
+            // we need to specify this parameter so webpack knows how to properly
+            // interpret it when encountered.
+            fs: 'empty'
+        },
+        output: Object.assign({}, config.output, {
+            library: [ 'JitsiMeetJS', 'app', 'effects' ],
+            libraryTarget: 'window',
+            filename: '[name].min.js',
+            sourceMapFilename: '[name].min.map'
+        }),
+        performance: getPerformanceHints(4 * 1024 * 1024)
```

Comment:
woha, 4MB! can you check where the size here comes from? the entire app is ~2.5 MB, and this doesn't include the wasm file...

Reply:
Aw sorry about that, I forgot to change this after I optimised the rnnoise binary for size. now It's under 20k so I changed the upper limit to 30 * 1024.